File: src/main/java/nl/dannyjelsma/soundboardbot/commands/RefreshCommand.java
package nl.dannyjelsma.soundboardbot.commands;

import com.jagrosh.jdautilities.command.Command;
import com.jagrosh.jdautilities.command.CommandEvent;
import nl.dannyjelsma.soundboardbot.audio.SoundManager;

public class RefreshCommand extends Command {

    public RefreshCommand() {
        name = "refresh";
        cooldown = 20;
    }

    @Override
    protected void execute(CommandEvent event) {
        SoundManager manager = SoundManager.getInstance();


        manager.clearSounds();
        manager.loadSounds();
        event.getChannel().sendMessage("Refreshed the sounds.").queue();
    }
}

File: src/main/java/nl/dannyjelsma/soundboardbot/commands/RefreshCommand.kt
package nl.dannyjelsma.soundboardbot.commands

import com.jagrosh.jdautilities.command.Command
import com.jagrosh.jdautilities.command.CommandEvent
import nl.dannyjelsma.soundboardbot.audio.SoundManager

class RefreshCommand : Command() {
    init {
        name = "refresh"
        cooldown = 20
    }

    override fun execute(event: CommandEvent) {
        SoundManager.clearSounds()
        SoundManager.loadSounds()
        event.channel.sendMessage("Refreshed the sounds.").queue()
    }
}